server/src/main/java/io/unitycatalog/server/persist/utils/FileUtils.java

```diff
@@ -47,6 +61,20 @@ public static String createTableDirectory(
     return createDirectory(absoluteUri).toString();
   }
 
+  public static String createRegisteredModelDirectory(
+      String catalogName, String schemaName, String modelName) {
+    String absoluteUri =
+        getModelDirectoryURI(catalogName + "." + schemaName + ".models." + modelName);
+    return createDirectory(absoluteUri).toString();
+  }
+
+  public static String createModelVersionDirectory(
```

Comment:
Will each version's location also be stored along with the model version entity ?

Reply:
Yes.  Every version has its own storage location which is computed off of the storage location of the registered model.